integrations/tensorflow/e2e/CMakeLists.txt

```diff
@@ -138,3 +138,5 @@ function(add_all_e2e_tests pattern)
 endfunction()
 
 add_all_e2e_tests("*_test.py")
+
+add_subdirectory(keras)
```

Comment:
We typically put `add_subdirectory(foo)` or `iree_add_all_subdirs()` at/near the top of each CMakeLists.txt, rather than at the bottom. Unless there is a specific reason (like needing some functions defined first) for putting this at the bottom, could you move it up?

Reply:
Nope, just didn't know!